### DISCUSSION TITLE
Prim Search in the Outliner

### DISCUSSION THREAD

**Discussion by brycegbrazen** [Autodesk/maya-usd#1832] [Q&A] [answered]
Hi everyone,

Does anyone know if it's possible or will be possible in the future to be able to search for USD prims in the outliner with the search bar at the top of the outliner? It seems that the search doesn't know about anything underneath the mayaUsdProxyShape node.

**Comment by seando-adsk** [comment, marked as answer]
Currently the Outliner search doesn't work for the the UFE items (USD prims).
